.github/workflows/ci.yaml

```diff
@@ -79,12 +79,9 @@ jobs:
           key: ${{ runner.os }}-vscode-1.70.2
 
       - name: Run E2E tests
-        uses: GabrielBB/xvfb-action@v1.0
+        uses: coactions/setup-xvfb@v1.0
```

Comment:
I'm not familiar with that extension, but it was also deprecated, so I replaced it with the one they suggested. The same maintainer is working on both. Anything else I should be checking?

Reply:
I don't think so. The `GabrielBB/xvfb-action` is borrowed as a pattern from microsoft vscode extension repos. If `coactions` is the replacement great.